### DISCUSSION TITLE
[Kavita] Ability to prioritize epub over pdf

### DISCUSSION THREAD

**Discussion by backyMacky** [Kareadita/Kavita#3130] [Ideas]
### Idea Description

When performing Search, that it would be great in case e.g. PDF is corrupted to continue looking for epub files - i have some examples where pdf is not working but epub is there and still it shows as "incomplete".

### Idea Category

User Experience

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
Why can't you just turn off PDF files via the library or use exclude patterns?

**Comment by ckuethe** [comment]
I see this occasionally with purchases from Humble Bundle; usually the books are available as EPUB and PDF, but sometimes a couple of items are only available as PDF.

I usually set my library content type to EPUB only, and create an additional PDF only library if I have to, but it would be nice if I could leave both formats enabled and have them merged together, with EPUB being preferred.
